bootstrap.sh

```diff
@@ -13,6 +13,9 @@ pipenv sync --dev
 # Run data migrations
 pipenv run python manage.py migrate
 
+# Install bootstrap icons
+pipenv install django-bootstrap-icons
```

Comment:
It was added automatically to Pipfile when I ran the command from the bootstrap script.
I want to include it in the bootstrap script so it will import the library to anyone who brings up the project.

Reply:
That's why you have `Pipfile`. It should be like django and pytest installation

Reply:
I removed the command from 'bootstrap.sh'.